stackit/internal/services/argus/instance/resource_test.go

```diff
@@ -993,16 +1077,43 @@ func TestToUpdateAlertConfigPayload(t *testing.T) {
 			},
 			isValid: true,
 		},
+		{
+			description: "no global options",
```

Comment:
Is this testing anything different for the global options than the test above? Maybe it could be `empty global options` and test the empty object